docs/SMC_Specification/Auxiliary_Management_Controller.md

```diff
@@ -414,35 +421,79 @@ If an SMC device provides power metrics, the EnvironmentMetric resource and Sens
-0x83 CHALLENGE | &nbsp; | Required
-0x84 GET_VERSION | &nbsp; | Required
-0xE0 GET_MEASUREMENTS | &nbsp; | Required
-&nbsp; | MEAS_CAP=10b | Required
-&nbsp; | DMTFSpecMeasurementValueType<br> - [00h] Immutable Rom<br> - [01h] Mutable FW | Required
-0xE1 GET_CAPABILITIES | &nbsp; | Required
-&nbsp; | CERT_CAP | Required
-&nbsp; | CHAL_CAP | Required
-&nbsp; | MEAS_CAP | Required
-0xE3 NEGOTIATE_ALGORITHMS | &nbsp; | Required
-&nbsp; | BaseAsymAlgo<br> - [Bit 2] TPM_ALG_RSASSA_3072 [CMA, CNSA,OCP] (Allowed)<br> - [Bit 4) TPM_ALG_ECDSA_ECC_NIST_P256[CMA] (Allowed)<br> - [Bit 7] TPM_ALG_ECDSA_ECC_NIST_P384[CMA, CNSA, OCP] (Preferred) | &nbsp;
-&nbsp; | BaseHashAlgo<br> - [Bit 0] TPM_ALG_SHA_256 [CMA] (Allowed)<br> - [Bit 1] TPM_ALG_SHA_384 [CMA, CNSA, OCP] (Preferred) | &nbsp;
-&nbsp; | MeasurementHashAlgo<br> - [Bit 1] TPM_ALG_SHA_256 [CMA] (Allowed)<br> - [Bit 2] TPM_ALG_SHA_384 [CMA, CNSA, OCP] (Preferred) | &nbsp;
-0xFF RESPOND_IF_READY | &nbsp; | Required
+0x01 DIGESTS | Required |  
+0x02 CERTIFICATE | Required |  
+0x03 CHALLENGE_AUTH | Required | Responder shall be able to provide the measurement summary type requested - <br> 0x0 - No measurement summary hash requested<br> 0x1 - TCB measurements only<br> 0xFF - All measurements
+0x04 VERSION | Required |  
+0x05 CHUNK_SEND_ACK | Optional |  
+0x06 CHUNK_RESPONSE | Optional |  
+0x60 MEASUREMENTS | Required | 
+0x61 CAPABILITIES | Required |  CERT_CAP - Responder shall support DIGESTS and CERTIFICATE response messages<br> CHAL_CAP - Responder shall support CHALLENGE_AUTH response message.<br> MEAS_CAP - Responder shall support MEASUREMENTS response and can generate signatures - MEAS_CAP=10b
+0x63 ALGORITHMS | Required | --**Implemenation Guidance**--<br><br> BaseAsymSel<br> - TPM_ALG_RSASSA_3072 [CMA, CNSA,OCP] (Allowed)<br> - TPM_ALG_ECDSA_ECC_NIST_P256[CMA] (Allowed)<br> - TPM_ALG_ECDSA_ECC_NIST_P384[CMA, CNSA, OCP] (Preferred)<br><br> BaseHashSel<br> - TPM_ALG_SHA_256 [CMA] (Allowed)<br> - TPM_ALG_SHA_384 [CMA, CNSA, OCP] (Preferred)<br><br> MeasurementHashAlgo<br> - TPM_ALG_SHA_256 [CMA] (Allowed)<br> - TPM_ALG_SHA_384 [CMA, CNSA, OCP] (Preferred) 
+0x64 KEY_EXCHANGE_RSP | Required |  
+0x65 FINISH_RSP | Required |  
+0x66 PSK_EXCHANGE_RSP | Optional |  
+0x67 PSK_FINISH_RSP | Optional |  
+0x68 HEARTBEAT_ACK | Optional |  
+0x6C END_SESSION_ACK | Required |  
+0x6D CSR | Recommended |  
+0x6E SET_CERTIFICATE_RSP | Recommended |  
+0x7E VENDOR_DEFINED_RESPONSE | Optional |  
+0x7F ERROR | Required |  
+
+### SPDM Certificate Requirements
+- The Root CA Trusted Certificate Authority shall be the device vendor and shall be the same across all device models developed by the device vendor.
+- Self-signed certificates are prohibited.
+- The AMC shall support the AliasCert or DeviceCert model in slot 0.
+  
+### SPDM Firmware Measurements
+- The AMC shall be able to generate signed measurements (i.e., MEAS_CAP field shall be set to 10b) and shall support the following Measurement block types:
```

Comment:
The "shall support" statement conflicts with the 'optional' and 'recommended' in the table, itself.